src/doc/dev_manual/test_suite.rst

```diff
@@ -185,7 +285,18 @@ Otherwise they will compare them as strings.
     Passes if bucket *contains* expected according to ``eqoper`` equality operator.
     Fails otherwise.
 
-For some examples, see `test_values_simple.py <https://github.com/visit-dav/visit/blob/develop/src/test/tests/unit/test_value_simple.py>`_.
+``TestFOA(name, tag='unk') :
+    Fail on arrival with test case outcome name the concatenation of ``name`` and ``tag``.
+    Whenever python execution arrives at a line with ``TestFOA()``, the test is considered a failure.
+    Typically, ``tag`` is ``LINE()`` to indicate the python line number where failure occured.
+    A given bit of test logic (e.g. a test *case*) can have *many* ``TestFOA()`` calls of the same ``name`` but with different ``tag``.
+    
+``TestPOA(name)``
+    Pass on arrival with test case outcome name just ``name``.
+    Whenever python execution arrives at a line with ``TestPOA()``, the test is considered a pass.
+    A given bit of test logic (e.g. a test *case*) can have *only one* ``TestPOA()`` call.
+
+For some examples, see `test_values_simple.py <https://github.com/visit-dav/visit/blob/develop/src/test/tests/unit/test_value_simple.py>`__ and `test_atts.py <https://github.com/visit-dav/visit/blob/develop/src/test/tests/unit/test_atts.py>`__.
```

Comment:
`test_atts.py` does not exist. Did you mean `atts_assign.py`?

Reply:
Yes, lemme fix right here...

```suggestion
For some examples, see `test_values_simple.py <https://github.com/visit-dav/visit/blob/develop/src/test/tests/unit/test_value_simple.py>`__ and `test_atts.py <https://github.com/visit-dav/visit/blob/develop/src/test/tests/unit/atts_assign.py>`__.
```